core/src/App.svelte

```diff
@@ -2126,7 +2125,6 @@
       on:handleSearchNavigation={handleSearchNavigation}
       bind:isSearchFieldVisible
       bind:displaySearchResult
-      bind:displayCustomSearchResult
```

Comment:
This seems to not be needed to me as well as it doesn't seem to be used.
But maybe @JohannesDoberer thinks it should stay for some reason, or is it just an 'outdated remove todo' from the past?

Reply:
@ndricimrr 
How do you come to the conclusion that `displayCustomSearchResult ` is not needed?
I can see it is used in TopNav and will be shared with GlobalSearch and GlobalSearchCentered or I'm completely wrong?

Reply:
That is true and in GlobalSearch(Centered) it seemed to do nothing, which is why I removed them. In one of them it came up as a Svelte warning, which is why I started looking. I think the question here is if I overlooked something and that prop does have a purpose.

Reply:
ah. Sorry my mistake.
Yes it can be removed.